test/turnstile_example_test.exs

```diff
@@ -0,0 +1,92 @@
+defmodule GenFSM.TurnstileExampleTest do
```

Comment:
We could move this into a different module. This naming was just the first that came to mind.

We also need to do more unit tests; the turnstile is more of an integration test.